app/src/main/java/org/lineageos/twelve/models/Audio.kt

```diff
@@ -28,7 +30,9 @@ data class Audio(
     val type: Type,
     val durationMs: Int,
     val artistUri: Uri,
+    val artist: String,
```

Comment:
I think we want to use the actual artist sting in areContentsTheSame to ensure we get updates if the title changes (while the URI should be fixed and just an id)

Reply:
> Or MediaProvider creates new uri for title change
Nope

Let's just check both just in case